stdlib/JuliaSyntaxHighlighting.version

```diff
@@ -1,4 +1,4 @@
 JULIASYNTAXHIGHLIGHTING_BRANCH = main
-JULIASYNTAXHIGHLIGHTING_SHA1 = 19bd57b89c648592155156049addf67e0638eab1
+JULIASYNTAXHIGHLIGHTING_SHA1 = eb8097c5f1dbfca80b5e6664af031ff1fe0904af
```

Comment:
When I was checking this commit, github gave a warning:
> This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

commit: https://github.com/JuliaLang/JuliaSyntaxHighlighting.jl/commit/eb8097c5f1dbfca80b5e6664af031ff1fe0904af

The change should be pushed into one of the `JuliaLang/JuliaSyntaxHighlighting.jl` branches.
Maybe `julia1-compat`?

Reply:
Ah yep, since this looks good to merge I think it's not presumptuous to just put that commit in the `main` branch now. I'll do so when I next sit down in front of my computer.